packages/cli/src/dbt/context.ts

```diff
@@ -5,6 +5,7 @@ import * as path from 'path';
 
 type GetDbtContextArgs = {
     projectDir: string;
+    mainProjectDir?: string;
```

Comment:
These changes belong to a separate PR that is already open. Should we remove them from this PR ? Otherwise the PR looks good 👍